src/test/analyzer_test.ts

```diff
@@ -168,6 +168,21 @@ suite('Analyzer', () => {
           ['polymer.html', 'foo.js', 'foo.css']);
     });
 
+    test('polymer css import scanner', async() => {
+      let contents = `<html><head>
+          <link rel="import" type="css" href="bar.css">
```

Comment:
Is this valid? I think we might only support that kind of import inside of a polymer template